idaes/config.py

```diff
@@ -56,81 +56,6 @@
     "DYLD_LIBRARY_PATH": os.environ.get("DYLD_LIBRARY_PATH", ""),
 }
 
-# Default configuration json string.  Store the default config like this because
```

Comment:
Just to be sure, was all this deletion expected?

Reply:
Yeah.  I moved the defaults in the Pyomo config block.